apps/settings/templates/settings/admin/server.php

```diff
@@ -71,7 +71,7 @@
 	   title="<?php p($l->t('Open documentation'));?>"
 	   href="<?php p(link_to_docs('admin-background-jobs')); ?>"></a>
 
-	<p class="settings-hint"><?php p($l->t('For optimal performance it\'s important to configure background jobs correctly. For bigger instances \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
+	<p class="settings-hint"><?php p($l->t('For Nextcloud to work correctly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```

Comment:
Please avoid labelling :)

```suggestion
	<p class="settings-hint"><?php p($l->t('For the server to work correctly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```

Reply:
fine by me :)

Reply:
done